ipatests/test_webui/test_cert.py

```diff
@@ -49,3 +132,392 @@ def test_read(self):
         rows = self.get_rows()
         self.navigate_to_row_record(rows[0])
         self.navigate_by_breadcrumb("Certificates")
+
+    @screenshot
+    def test_search_subject(self):
+        """
+        Try to search certificate by subject
+        """
+        self.init_app()
+        self.navigate_to_entity(ENTITY)
+        self.select('select[name=search_option]', 'subject')
+        search_pkey(self, 'Certificate Authority')
+        rows = self.get_rows()
+        assert len(rows) != 0
+
+        # try to search non-existent subject
+        self.navigate_to_entity(ENTITY)
+        self.select('select[name=search_option]', 'subject')
+        search_pkey(self, 'nonexistent')
+        rows = self.get_rows()
+        assert len(rows) == 0
+
+        # try to search subject with speacial char
+        self.navigate_to_entity(ENTITY)
+        self.select('select[name=search_option]', 'subject')
+        search_pkey(self, '<,>.?/')
+        rows = self.get_rows()
+        assert len(rows) == 0
+
+        # try to search subject with leading space
+        check_space_error(self, ' Certificate Authority', 'subject')
+
+        # try to search subject with trailing space
+        check_space_error(self, 'Certificate Authority ', 'subject')
+
+    @screenshot
+    def test_search_revocation_reason(self):
+        """
+        Try to search certificates by revocation reason
+        """
+        self.init_app()
+        self.navigate_to_entity(ENTITY)
+
+        # add a new cert
+        record = add_cert(self, 'HTTP/master.testrelm.test', CERT_CSR)
```

Comment:
We do not have "master.testrelm.test" host yet so the test will fail (and is failing in PRCI indeed)

Reply:
Looks like other 2 fails in PRCI are consequence of the first one.